scripts/assetcheck.sh

```diff
@@ -1,11 +1,7 @@
 #!/usr/bin/env bash
 
-if [[ ! -e http/bindata_assetfs.go ]]
+if [[ ! -e http/web_ui/index.html ]]
 then
   printf "Compiled UI assets not found. They can be built with: make static-dist\n\n"
-else
```

Comment:
Why was this else block removed?

Reply:
It went on to specifically reference the generated bindata file:

https://github.com/hashicorp/vault/blob/6b2f99cd6e1c0bb654c72ac7459f7956a429f927/scripts/assetcheck.sh#L6-L11

I didn't find it useful. We could restore it and test `index.html` I suppose?

Reply:
Sorry for the delay. But yes, having a check against a representative generated file like `index.html` would still be useful. We added this after a number of cases of people digging into and reporting some odd UI-related behavior, only to find they were running very out of date UI code.